app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,318 @@
+
+If you believe you meet the eligibility requirements, but have not been informed by your teacher training provider that you may be awarded the IRP, please contact your training provider directly first. If your training provider cannot help you, you can email us at teach.inengland@education.gov.uk. 
+
+### Salaried trainees 
+
+If you have a place on a teacher training course where you are paid a salary to work while you train, email us at IRP.ExpressInterest@education.gov.uk to let us know you are interested in applying for the IRP. You will be asked to fill in a form giving us your details and we’ll get in touch when IRP applications open for salaried trainees in autumn 2023. If you are eligible, you’ll get the IRP around the end of your first term. 
+
+## The international relocation payment for teachers  
+
+To be eligible for the international relocation payment (IRP), you must meet all 3 of the following requirements.
+
+### 1. Teaching subject
+
+You must have accepted a job as a teacher of languages or physics in an English state secondary school on a contract lasting at least one academic year.  
+
+a) Teachers of all languages offered in English state secondary schools are eligible to apply for the IRP. The language or languages can be combined with another subject, but must make up at least 50% of teaching time. You can combine teaching English with another language, as long as the non-English language makes up at least 50% of your teaching timetable.  
+
+b) Physics can be taught with another subject, but must make up at least 50% of teaching time. Teachers of general science or combined science  are also eligible to apply for the IRP, as long as they are teaching the physics element of general science or combined science. General science or combined science can be taught with another subject, but must make up at least 50% of teaching time.  
+
+c) 'State schools' are schools funded by the UK government. 'Secondary schools' teach children aged 11 to 16 or 11 to 18. You will not be eligible if you teach in a school for pupils aged 16 to 18 only.  
+
+d) An academic year is 3 full school terms: for example, if your teaching job started in September 2023 (the start of the autumn term), it would need to continue until July 2024 (covering the spring and summer terms). 
+
+[Check the type and age range of the school](https://www.get-information-schools.service.gov.uk/) where you’ll be teaching. You can also find out about the [different types of state school in England](https://www.gov.uk/types-of-school). 
+
+### 2. Type of visa
+
+You must come to England on one of the visas listed here.  
+ 
+For the following visas, you must have applied for your visa from outside the UK:  
+
+* [Skilled worker visa](https://www.gov.uk/skilled-worker-visa) 
```

Comment:
given that many of these visas are the same for teachers/trainees, would it not make sense to combine these two sections and just have one section on visas? then if you need to clarify any, for example, the student visa, you could just put extra detail after the visa in brackets, like so:

'Student visa (if you're training to teach in England)'